dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanner.cs

```diff
@@ -57,16 +57,15 @@ public async Task<Plan> CreatePlanAsync(string goal, CancellationToken cancellat
             throw new SKException("The goal specified is empty");
         }
 
-        string relevantFunctionsManual = await this._context.GetFunctionsManualAsync(goal, this.Config, cancellationToken).ConfigureAwait(false);
-        this._context.Variables.Set("available_functions", relevantFunctionsManual);
+        string relevantFunctionsManual = await this._kernel.CreateNewContext().GetFunctionsManualAsync(goal, this.Config, cancellationToken).ConfigureAwait(false);
```

Comment:
Not related to this PR but I notice that the `SequentialPlanner` is the only planner that uses an extension method off of `SKContext` to get the available functions whereas the others use private methods on the planner classes. Moving the shared logic for this would be another opportunity to improve consistency of behavior between planners (not to mention make it easier to maintain).

Reply:
This is a temporary workaround... with Roger's change, we should no longer need CreateNewContext() for anything.